website/docs/d/cloud_provider_access.markdown

```diff
@@ -10,14 +10,14 @@ description: |-
 
 # Data Source: mongodbatlas_cloud_provider_access
 
-`mongodbatlas_cloud_provider_access` allows you to get the list of cloud provider access roles, currently only AWS is supported.
+`mongodbatlas_cloud_provider_access` allows you to get the list of cloud provider access roles, currently only AWS and Azure is supported.
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
 
 ## Example Usage
```

Comment:
Can you add an Azure example as well under Example Usage?

Reply:
I removed this change as `mongodbatlas_cloud_provider_access` does not support azure. (this was a left-over from before we discussed the issue with the single path)

regarding this Azure example: I added the azure example here https://github.com/mongodb/terraform-provider-mongodbatlas/tree/INTMDB-545/examples/atlas-cloud-provider-access/azure. is there something you want me to add to the example?

Reply:
no concerns, thx!